components/brave_ads/core/internal/creatives/new_tab_page_ads/creative_new_tab_page_ad_wallpapers_database_table.cc

```diff
@@ -23,6 +24,20 @@ namespace {
 
 constexpr char kTableName[] = "creative_new_tab_page_ad_wallpapers";
 
+std::string ConditionMatchersToString(
+    const NewTabPageAdConditionMatchers& condition_matchers) {
+  std::vector<std::string> condition_matchers_as_string;
+  condition_matchers_as_string.reserve(condition_matchers.size());
+
+  for (const auto& [pref_name, pattern] : condition_matchers) {
+    const std::string condition_matcher =
+        base::StrCat({pref_name, "=", pattern});
```

Comment:
Can we add an if statement or CHECK for pattern doesn't contain `=` and `;` characters?